x-pack/plugins/security_solution/public/timelines/components/timeline/body/helpers.tsx

```diff
@@ -129,51 +123,6 @@ export const getEventType = (event: Ecs): Omit<TimelineEventsType, 'all'> => {
   return 'raw';
 };
 
-export const isInvestigateInResolverActionEnabled = (ecsData?: Ecs) =>
```

Comment:
moved to `.../detections/components/alerts_table/timeline_actions/investigate_in_resolver.tsx`